src/components/panel/panel.js

```diff
@@ -1306,10 +1386,22 @@ MdPanelRef.prototype._updatePosition = function(init) {
       this.panelContainer.addClass(MD_PANEL_HIDDEN);
     }
 
-    this.panelEl.css(MdPanelPosition.absPosition.TOP, positionConfig.getTop());
-    this.panelEl.css(MdPanelPosition.absPosition.BOTTOM, positionConfig.getBottom());
-    this.panelEl.css(MdPanelPosition.absPosition.LEFT, positionConfig.getLeft());
-    this.panelEl.css(MdPanelPosition.absPosition.RIGHT, positionConfig.getRight());
+    this.panelEl.css(
```

Comment:
@EladBezalel @ErinCoughlan has informed me that her syntax conventions are pretty strict on lines exceeding the 80 character soft-wrap line. So, as a part of my fix for #9357, I went through and made those changes.